package.json

```diff
@@ -39,7 +39,6 @@
     "eslint-plugin-import": "2.3.0",
     "flow-bin": "0.38.0",
     "jasmine-core": "2.5.2",
-    "jshint": "2.9.4",
```

Comment:
Yes. It's provided by Temasys. We are updating it from time to time. It's interesting why it is part of our source code but unfortunately I don't know. 

Anyway this is whole different story :)